src/algorithms/wcc.c

```
@@ -0,0 +1,170 @@
+
+	// enforece labels
+	if(n_lbls > 0) {
+		Delta_Matrix DL = Graph_GetLabelMatrix(g, lbls[0]);
+
+		GrB_Matrix L;
+		info = Delta_Matrix_export(&L, DL);
+		ASSERT(info == GrB_SUCCESS);
+
+		// L = L U M
+		for(unsigned short i = 1; i < n_lbls; i++) {
+			DL = Graph_GetLabelMatrix(g, lbls[i]);
```

Cppcheck (2.10-2)

[error] Memory leak

(memleak)